db/migrations/20221117154600-add-new-clientn-key-column-to-installation-table.js

```diff
@@ -0,0 +1,16 @@
+"use strict";
```

Comment:
Thinking out loud... Do we want to seperate this migration as a seperate step? I think since its a new endpoint thats being called we are probably ok? but just threw it out for consideration

Reply:
Haha, yes, YES!!
Therefore I put this PR back to draft. Wait for another one to go first. 
Good pickup.